packages/manager/modules/pci/src/projects/project/storages/volume-backup/volume-backup.routing.js

```diff
@@ -24,6 +24,29 @@ export default /* @ngInject */ ($stateProvider) => {
           'pci_projects_project_storages_volume_backup_breadcrumb',
         ),
 
+      guides: /* @ngInject */ (initGuides) => initGuides(GUIDES),
+
+      initGuides: /* @ngInject */ (coreConfig, $translate) => (guides) => {
+        return guides.reduce(
+          (list, guide) => [
+            ...list,
+            {
+              ...guide,
+              title: $translate.instant(
+                `pci_projects_project_storages_volume_backup_guides_${guide.id}_title`,
+              ),
+              description: $translate.instant(
+                `pci_projects_project_storages_volume_backup_guides_${guide.id}_description`,
+              ),
+              link:
+                guide.links[coreConfig.getUser()?.ovhSubsidiary] ||
+                guide.links.DEFAULT,
+            },
+          ],
+          [],
+        );
+      },
```

Comment:
**nitpick:** `initGuides` can be avoided since it is not used in the component.

Reply:
done